patches/open+7.0.2.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/open/index.js b/node_modules/open/index.js
+index 5fe49ad..c9194ba 100644
+--- a/node_modules/open/index.js
++++ b/node_modules/open/index.js
+@@ -10,7 +10,9 @@ const pAccess = promisify(fs.access);
+ const pExecFile = promisify(childProcess.execFile);
+ 
+ // Path to included `xdg-open`.
+-const localXdgOpenPath = path.join(__dirname, 'xdg-open');
++const localXdgOpenPath = process.pkg
++	? path.join(path.dirname(process.execPath), 'xdg-open')
++	: path.join(__dirname, 'xdg-open');
```

Comment:
This patch is the key fix to the issue.

Reply:
Works perfectly, as tested on Ubuntu 18.04.